packages/cerebral/src/operators/helpers/populatePath.js

```diff
@@ -5,6 +5,10 @@ export default function populatePath (context, strings, values) {
       return currentPath + string + context.state.compute(valueTemplate)
     }
 
-    return currentPath + string + (valueTemplate ? valueTemplate(context).toValue() : '')
+    if (valueTemplate && typeof valueTemplate === 'function') {
+      return currentPath + string + valueTemplate(context).toValue()
+    }
+
+    return currentPath + string + (valueTemplate ? valueTemplate : '')
```

Comment:
ok thanks,  i've used ... + (valueTemplate || '') now. (in () because of + gets executed before || and we just wanna make sure that valueTemplate has a value)
need bigger linting symbols ;)